Cargo.toml

```diff
@@ -80,8 +80,8 @@ tokio                      = "1"
 toml                       = "0.8"
 tracing                    = "0.1"
 tracing-subscriber         = "0.3"
-wasmer                     = "5"
-wasmer-middlewares         = "5"
+wasmer                     = "=5.0.0"
+wasmer-middlewares         = "=5.0.0"
```

Comment:
With wasmer 5.0.1 we get an error when building dango-cli:

```
error: failed to run custom build command for `cranelift-codegen v0.110.2`

Caused by:
  process didn't exit successfully: `/Users/larry/workspace/left-curve/left-curve/target/release/build/cranelift-codegen-725296997bf7694e/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-changed=build.rs

  --- stderr
  Error: directory isle_generated_code/ exists but is only used when
  `--feature isle-in-source-tree` is specified. To prevent confusion,
  this build script requires the directory to be removed when reverting
  to the usual generated code in target/. Please delete the directory and
  re-run this build.
```

Reply:
We can fix this by deleting the folder:

```
~/.cargo/registry/src/index.crates.io-{hash}/cranelift-codegen-{version}/isle_generated_code
```

No need to fix the version to `=5.0.0`.